nextcord/abc.py

```diff
@@ -1531,33 +1532,6 @@ async def fetch_message(self, id: int, /) -> Message:
         data = await self._state.http.get_message(channel.id, id)
         return self._state.create_message(channel=channel, data=data)
 
-    async def pins(self) -> List[Message]:
-        """|coro|
-
-        Retrieves all messages that are currently pinned in the channel.
-
-        .. note::
-
-            Due to a limitation with the Discord API, the :class:`.Message`
-            objects returned by this method do not contain complete
-            :attr:`.Message.reactions` data.
-
-        Raises
-        -------
-        ~nextcord.HTTPException
-            Retrieving the pinned messages failed.
-
-        Returns
-        --------
-        List[:class:`~nextcord.Message`]
-            The messages that are currently pinned.
-        """
-
-        channel = await self._get_channel()
-        state = self._state
-        data = await state.http.pins_from(channel.id)
-        return [state.create_message(channel=channel, data=m) for m in data]
-
```

Comment:
this'll want to be added to the migrating_2.rst as it is a breaking change (people relying on `isinstance`ing a messageable and expecting pins)